commit ee663130c96dc7140a97d524781477742b17ce68 (refs/changes/62/3962/1)
Author: sjorge <sjorge@blackdot.be>
Date:   2018-05-16T21:58:48+02:00 (1 year, 5 months ago)
    
    OS-XXXX add_fbuf should also use is_env_string

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index 701ad16c83..c6431e61f6 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -486,8 +486,7 @@ add_fbuf(int *argc, char **argv)
 	/*
 	 * Do not add a frame buffer or tablet if VNC is disabled.
 	 */
-	if ((val = get_zcfg_var("attr", "vnc_port", NULL)) != NULL &&
-	    strcmp(val, "-1") == 0) {
+	if (is_env_string("attr", "vnc_port", NULL, "-1")) {
 		return (0);
 	}
 
